VotingApplication/VotingApplication.Web/Scripts/Directives/VoteChoice/VoteChoiceTypes/PointsVoteChoice.js

```diff
@@ -0,0 +1,76 @@
+﻿(function () {
+    'use strict';
+
+    angular
+        .module('VoteOn-Vote')
+        .directive('pointsVoteChoice', pointsVoteChoice);
+
+    function pointsVoteChoice() {
+        return {
+            restrict: 'E',
+            scope: {
+                choices: '=',
```

Comment:
They're two-way binds to the parent objects so that it can modify the `VoteValue` property and update if the max points change.